README.md

```diff
@@ -46,6 +46,21 @@ To run the performance tests locally, follow these steps:
 
 This command will initiate the performance tests using Gatling and generate a report upon completion.
 
+4. Setup Git Hook for Code Formatting  
+   To automatically format code before committing, set up the Git hook:  
+
+   ```sh
+   cp pre-commit .git/hooks/pre-commit
+   chmod +x .git/hooks/pre-commit
+   ```
```

Comment:
Git tracks file permissions. Therefore you can make the file executable and commit it with this PR. So that developers only have to copy it and no need to make it executable again.

There's another way to do the same,
- create a directory called .githooks/pre-commit
- make the script executable
- commit the file and update the PR

Then, mention on the readme asking developers to run the following:
```
git config core.hooksPath .githooks
```

Reply:
@jayasanka-sack  thanks for review. I have update my PR